doc/user/example_aliases2.tex

```diff
@@ -11,17 +11,17 @@
     every join/.style={link}
 ]
 
-\node[file, on chain] (s3-current) {current/m012/u.alias};
-\node[file, on chain, join] (s3-config) {config/m012/u/rx17.alias};
+\node[file, on chain] (s3-current) {current/individual/m012/u.alias};
+\node[file, on chain, join] (s3-config) {config/individual/m012/u/rx17-2021.alias};
 \node[file, on chain] (s3-fixed) {fixed/sha256_0ae\ldots b9.h5};
 \node[file, on chain=going left, join=with s3-config] (s3-fixed-new)
   {fixed/sha256_d71\ldots 0a.h5};
 \node[fit={(s3-current) (s3-config) (s3-fixed) (s3-fixed-new)},
       draw, label={Model store}] {};
 
 \node[right=of s3-config, start chain=ts going below, on chain] (ts-latest) {
-    \ldots m012_model_primary_beam_config};
-\node[on chain=ts] (ts-fixed) {\ldots m012_model_primary_beam_fixed};
+    \ldots model_primary_beam_individual_config[m012]};
+\node[on chain=ts] (ts-fixed) {\ldots model_primary_beam_individual_fixed[m012]};
```

Comment:
So "fixed" really means "the actual model that any component would get during the observation if they asked for it". What if no component asked for it? Say you imaged a week later, and the imager is the only component using a PB. In this case the "fixed" key is sort of useless.

But I guess if something did use it at the time, and you wanted to make sure you used the same model throughout, then it is a good idea to fix it beforehand, instead of asking each component what it had used (which may also then differ between components).

Reply:
> So fixed really means "the actual model that any component would get during the observation if they asked for it".

Yes.

> But I guess if something did use it at the time, and you wanted to make sure you used the same model throughout, then it is a good idea to fix it beforehand, instead of asking each component what it had used (which may also then differ between components).

That's my main motivation for having the master controller export this key, rather than individual components doing their own things.

> And the fixed key has to be written to telstate at observe time, so future imaging etc would then need to store their actual model keys somewhere else, I guess. Or would that metadata also go into telstate?

It probably depends what's doing the imaging. If it's the MeerKAT automated pipeline it can dump information into telstate. If it's some after-market imager (e.g. something running at IDIA) it's doing to have to make its own arrangements. For katsdpimager I would probably try to record it as HISTORY/COMMENT in the FITS file anyway.

Reply:
Thanks, that is helpful to know.

I'm now shifting my focus to ska-sa/katsdpmodeldata#5 but leaving this open, given that comments there might affect the design here.